index.html

```diff
@@ -55,8 +55,45 @@
       }
     </style>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
-    <script>
-      // Put some code in here!
+        <script>
+    var countDown = 1234;
+    //test//
+    // $("#counter").val(countDown);
+
+    function Start(){
+      
+      console.log("I got clicked");
+      $("#counter").val(countDown);
+      setInterval(Counter(), 1000);
+    };
+
+
+    function Counter(){
+      $("#counter").val(countDown);
+      console.log("TEST "+countDown);
+
+      countDown = countDown - 1;
+    };
+
+$("#counter").click(Start());
```

Comment:
Same comment as the setInterval code above.  We want to call the `Start` function when `#counter` is clicked.  We don't want to pass the result of the function being executed.  `$("#counter").click(Start);`.